src/content/pages/tickets.mdx

```diff
@@ -89,17 +89,22 @@ Regular tickets are available until **21 June 2024**, after which Late-bird rate
 
 Education ticket won't have late bird variants.
 
-{/*
-<!-- ### Volume Discount
+### Volume Discounts
+
+Is your company attending the conference as a team? We offer the following volume discount:
 
- Is your company attending the conference as a team? We offer the following volume discount (Early Bird tickets excluded):
+- get 5 business tickets of any type for the price of 4
+- get 10 business tickets of any type for the price of 8
+- get 15 business tickets of any type for the price of 11
 
-  - get 5 business tickets of any type for the price of 4
-  - get 10 business tickets of any type for the price of 8
-  - get 15 business tickets of any type for the price of 11
+Note:
 
- Note: Volume discount will not apply if you mix & match between different ticket types. -->
+- Volume discount will not apply if you mix & match between different ticket tiers (e.g. business with personal).
```

Comment:
This is correct, but maybe we can simplify by just explicitly saying it only applies only to the business tickets.